packages/wallets/docs/examples/FlowProvider/AccountFlow/AccountFlow.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { ModalStepWrapper } from '../../../../src/components/Base/ModalStepWrapper';
+import { FlowProvider, TFlowProviderContext, useFlow } from '../../../../src/components/FlowProvider';
+import { useModal } from '../../../../src/components/ModalProvider';
+import { MT5AccountType } from '../../../../src/features/cfd/screens/MT5AccountType';
+import VerificationFlow from './VerificationFlow';
+
+const PasswordScreen = () => {
+    return (
+        <div style={{ display: 'grid', fontSize: 40, height: '100%', placeItems: 'center', width: '100%' }}>
+            Password Screen in Account Flow
+        </div>
+    );
+};
+
+const ScreenB = () => {
+    const { formValues, setFormValues } = useFlow();
+    return (
+        <div style={{ display: 'flex', flexDirection: 'column', gap: '4px', height: '30vh', padding: '4px' }}>
+            <h1>Screen B</h1>
+            <input
+                onChange={e => setFormValues('testb', e.target.value)}
+                style={{ border: '1px solid black', padding: '4px', width: '100%' }}
+                type='text'
+                value={formValues.testb}
+            />
+            <input
+                onChange={e => setFormValues('testa', e.target.value)}
+                style={{ border: '1px solid black', padding: '4px', width: '100%' }}
+                type='text'
+                value={formValues.testa}
+            />
+        </div>
+    );
+};
+
+const ScreenA = () => {
+    const { formValues, setFormValues } = useFlow();
+
+    return (
+        <div style={{ height: '30vh', padding: '4px' }}>
+            <input
+                onChange={e => setFormValues('testa', e.target.value)}
+                style={{ width: '100%', border: '1px solid black', padding: '4px' }}
+                type='text'
+                value={formValues.testa}
+            />
+        </div>
+    );
+};
+
+const JurisdictionScreen = () => {
+    return (
+        <div>
+            <h1>Jurisdiction Screen</h1>
+        </div>
+    );
+};
+
+const screens = {
+    aScreen: <ScreenA />,
```

Comment:
generally relying on order of keys in object is not good practice (best to my knowledge majority of modern browsers do keep the keys of object in order, but still its not good practice to rely on this non-explicit behaviour)